keepassxc-browser/content/fields.js

```diff
@@ -95,9 +95,23 @@ kpxcFields.getSegmentedTOTPFields = function(inputs, combinations) {
 
     let exceptionFound = false;
 
+    // Returns true if all 6 inputs are numeric, tel or text/number with maxLength of 1.
+    // If ignoreLength is true, number of TOTP fields are allowed to differ from the default (6).
+    const allAreSegmentedFields = (inputs, ignoreLength = false) =>
```

Comment:
This is better but recommend a variable rename (also in the main function below) to: `ignoreFieldCount`. This function should probably also be: `areFieldsSegmentedTotp` or similar. Personally, I think we should still set a count limit when ignoring the field count, TOTP will never be less than 4 fields.

Reply:
Done.